### PR TITLE
#FIX-25 translation to Swedish

### PR DESCRIPTION
**Pull request by chakra2017**

### CODE REVIEW

**Comment by zer0Kerbal** [comment]
tack @chakra2017 !
closes #25 Swedish (Svenska) <sw.cfg>

File: GameData/Mkerb/MkerbInc/Localization/sw-sw.cfg [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+		#MKERB-Sound-ApturSrfLanded = The microphone is working.
+
+		#MKERB-Sound-LaveInSpaceHigh = You hear a strange sqeek noise.
+		#MKERB-Sound-LaveSrfLanded = The microphone got wet.
+
+		#MKERB-Sound-TitanusInSpace = The device freaks out. Maybe jeb did play with it.
+		#MKERB-Sound-TitanusSrfLanded = You hear some movement under the ground.
+
+		#MKERB-Sound-VanorInSpace = The dust in the device makes the data unreliable.
+		#MKERB-Sound-VanorSrfLanded = It seems to pick up some data.
```
Comment (zer0Kerbal) [inline GameData/Mkerb/MkerbInc/Localization/sw-sw.cfg:456]: @chakra2017 would you also translate these strings?

Comment (chakra2017) [reply GameData/Mkerb/MkerbInc/Localization/sw-sw.cfg:456]: ok

**Review by zer0Kerbal — APPROVED** [review]
Tack @chakra2017 
closes #25

**Pull request merged**